Jenkinsfile.oscontainer

```diff
@@ -0,0 +1,23 @@
+def DOCKER_IMG = "registry.fedoraproject.org/fedora:28"
+// Turns out the Jenkins docker stuff totally breaks with SELinux
+def DOCKER_ARGS = "--privileged"
+
+node(env.NODE) {
+    checkout scm
+
+    stage("Prepare Dockerfile") {
+        docker.image(DOCKER_IMG).pull()
+        docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+          sh """sed -e 's,@OSTREE_REPO_URL@,${env.OSTREE_INSTALL_URL},' < Dockerfile.rollup.in > Dockerfile.rollup"""
+        }
+    }
+    docker.withRegistry('https://docker-registry.engineering.redhat.com', '46060330-e169-4fd9-a135-64d92a52968a') {
```

Comment:
Minor, for the credential, do we want to show it publicly?

Reply:
Eh, it's just a random [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier).

Reply:
Ah, srry, I got confused a earlier. I thought this UUID represents the credentials :p. And I saw this:

>For a Docker Registry which requires authentication, add a "Username/Password" Credentials item from the Jenkins home page and use the Credentials ID as a second argument to withRegistry()

Makes sense, thanks =).

Reply:
Hmm, can we pass them through parameters instead? To try to not bake internal URLs, but also to make e.g. pointing at a different registry easier.

Reply:
OK...done.  Though I haven't tested the result as I haven't yet got the activation energy up to re-setup jjb locally and figure out what you did to run it and change the jobs to use a local one etc.